operations/research-and-development/engineering/cloud-infrastructure-cost-kpis.md

```diff
@@ -0,0 +1,77 @@
+# Cloud Infrastructure Cost KPIs
+
+The Cloud SRE team uses certain KPIs to monitor cloud infrastructure costs from the engineering standpoint. These KPIs are used to help the business and the Cloud SRE team to set goals for cost optimization.
+
+As of July 2021, the goals for the SRE team are to
+
+ - Decrease the Average Cost Per Workspace (APWC)
+ - Keep development and test environments costs (DM) below 20% of total costs
+ - Increase the WB fraction (ratio of paid workspaces relative to total workspaces)
+
+Below are the primary and seconday KPIs used to measure each:
+
+## Primary KPIs
+
+### Average Production Workspace Cost
+
+ - **Description**: The fraction which compares the amount of Production Environment costs with the number of workspaces which at a given month of the Cloud Infrastructure service.
```

Comment:
Stavros & Isabel to discuss and clarify what is captured in production environment costs.

Reply:
@jasonblais I agree.

Reply:
Discussed with Stavros. I think after the first sentence in the Description, we can add "Production Environment will be defined the environment where we host all our customers, paid and free workspaces". Just to give a general idea of what the prod environment is. Feel free to add more description if necessary to add to this definition.

Reply:
Thanks! Added